### ISSUE TITLE
Inconsistent close buttons

### ISSUE THREAD

**Issue by stevage** [TerriaJS/terriajs#2090]
Apparently this week is open season on the designers? :) I have no idea if this matters, but:

![screenshot 2016-09-16 11 44 58](https://cloud.githubusercontent.com/assets/678344/18572987/40560e4e-7c03-11e6-9164-fcdc33ef13e1.png)

![screenshot 2016-09-16 11 44 50](https://cloud.githubusercontent.com/assets/678344/18572988/4080779c-7c03-11e6-8e3e-e4f5c77c8dce.png)

![screenshot 2016-09-16 11 44 46](https://cloud.githubusercontent.com/assets/678344/18572989/40843422-7c03-11e6-9761-3f6bbd5bdef0.png)

![screenshot 2016-09-16 11 44 44](https://cloud.githubusercontent.com/assets/678344/18572990/40a9a7d4-7c03-11e6-887f-5aef8ac83213.png)

**Comment by RacingTadpole** [comment]
And also from memory that "Your system seems to be running slowly, switching to 2D" box has no close X, only an OK button. Would an X as well be helpful? Generally I expect people just want to get rid of the warning quickly.

**Comment by chloeleichen** [comment]
@philipgrimmett

**Comment by chloeleichen** [comment]
i think they should all be X

**Comment by stevage** [comment]
Yep, and allow clicking outside the box to also close it, wherever that makes sense.

**Comment by philipgrimmett** [comment]
Subtle differences for each case:
- Image 1: Dropdown elements generally don't require a close button (clicking away is more common)
- Image 2: Adding/browsing for data is a task to be completed, hence 'done'. Using this wording frames the action, and hopefully gives the user more confidence while toggling between the main map and the catalogue.
- Image 3: I think the 'X' close is appropriate here. When a user open and closes feature info, the states of the map remains unchanged. 'Done' might suggest otherwise.
- Image 4: hmm. We tried to match the '+' add action with a '-' remove action, but looking back at it now I don't think it really works. At the risk of contradicting my point on Image 3, I think an 'X' icon or rubbish bin icon alongside the 'remove' text would be more recognisable for the average user.

Thoughts?

**Comment by stevage** [comment]
Yep. Specifically, yeah a rubbish bin (I didn't think of that) or X would be better for the Remove action. 

And I get your reasoning for Image 2, but there seems something slightly strange about putting the Done up in the top right. It's sort of like combining the "end of workflow" action with the "close window" action and yielding what is IMHO an odd result.

**Comment by RacingTadpole** [comment]
Note also this request to change the Data Catalog's Done button to X: https://github.com/TerriaJS/neii-viewer/issues/89.

**Comment by kring** [comment]
GA requested that the "Done" button on the location search panel be changed to "Clear" or "Clear Results" in TerriaJS/nationalmap#435.  I don't think I agree, but wanted to note it here for @philipgrimmett's consideration.

**Comment by tkeuneman** [comment]
Suggestions:

1. Related Maps expand: The only thing annoying about this is that if I click away, it thinks I'm trying to select a location. I then get a 'Feature Information No data is available here - try another location.' message that I then have to dismiss, so it ends up being two clicks to dismiss.
_Suggestion:_ Keep this dismissible on click, but make it so that if this box is open, and I click somewhere else, it ignores the "find data in this location" function.

<img width="1152" alt="screen shot 2018-03-05 at 11 28 57 am" src="https://user-images.githubusercontent.com/2201814/36952792-73ea2258-2068-11e8-8449-fd19bce1a543.png">

2. Data Catalogue search: Done is good. I'd suggest to change this to 'Apply', but it's a personal preference.

<img width="773" alt="nationalmap" src="https://user-images.githubusercontent.com/2201814/36954523-cdb7e690-2076-11e8-8be4-086718004810.png">

3. Agree that we should keep this as an X
<img width="439" alt="screen shot 2018-03-05 at 1 33 43 pm" src="https://user-images.githubusercontent.com/2201814/36955013-e04a0114-2079-11e8-906d-c5745b906285.png">

4. Remove and Remove all on left hand nav: I'd remove the minus icons and wouldn't introduce any other icons, choosing to keep these as text only. I'd argue that the other icons are for main call to actions/buttons, and the remove is more subtle than a primary button. 

<img width="343" alt="screen shot 2018-03-05 at 1 20 50 pm" src="https://user-images.githubusercontent.com/2201814/36954770-535cd606-2078-11e8-9875-40fe83758cb4.png">
